File: app_manager/appmanager/src/main/java/app/editors/manager/ui/fragments/main/AboutFragment.java
package app.editors.manager.ui.fragments.main;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.RelativeLayout;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.appcompat.widget.AppCompatTextView;

import app.editors.manager.BuildConfig;
import app.editors.manager.R;
import app.editors.manager.ui.fragments.base.BaseAppFragment;
import butterknife.BindView;
import butterknife.ButterKnife;
import butterknife.OnClick;
import butterknife.Unbinder;
import lib.toolkit.base.managers.utils.FileUtils;

public class AboutFragment extends BaseAppFragment {

    public static final String TAG = AboutFragment.class.getSimpleName();
    public static final String ASSETS_SDK_VERSION_PATH = "sdk.version";

    protected Unbinder mUnbinder;
    @BindView(R.id.about_app_version)
    protected AppCompatTextView mAboutAppVersion;
    @BindView(R.id.about_terms)
    protected RelativeLayout mAboutTerms;
    @BindView(R.id.about_policy)
    protected RelativeLayout mAboutPolicy;
    @BindView(R.id.about_license)
    protected RelativeLayout mAboutLicense;
    @BindView(R.id.about_website)
    protected RelativeLayout mAboutWebsite;

    public static AboutFragment newInstance() {
        return new AboutFragment();
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        final View view = inflater.inflate(R.layout.fragment_about, container, false);
        mUnbinder = ButterKnife.bind(this, view);
        return view;
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        init(savedInstanceState);
    }

    @Override
    public void onDestroyView() {
        super.onDestroyView();
        mUnbinder.unbind();
    }

    @OnClick({ R.id.about_terms,
                     R.id.about_policy,
                     R.id.about_license,
                     R.id.about_website})
    protected void onItemClick(final View view) {
        switch (view.getId()) {
            case R.id.about_terms:
                showUrlInBrowser(getString(R.string.app_url_terms));
                break;
            case R.id.about_policy:
                showUrlInBrowser(getString(R.string.app_url_policy));
                break;
            case R.id.about_license:
                showFragment(LicenseFragment.newInstance(), LicenseFragment.TAG, false);
                break;
            case R.id.about_website:
                showUrlInBrowser(getString(R.string.app_url_main));
                break;
        }
    }

    private void init(final Bundle savedInstanceState) {
        setActionBarTitle(getString(R.string.about_title));

        final String sdkVersion = FileUtils.readSdkVersion(requireContext(), ASSETS_SDK_VERSION_PATH);

        mAboutAppVersion.setText(getString(R.string.about_app_version, BuildConfig.VERSION_NAME,
                String.valueOf(BuildConfig.VERSION_CODE), sdkVersion));
    }

}

File: app_manager/appmanager/src/main/java/app/editors/manager/ui/fragments/main/AboutFragment.kt
package app.editors.manager.ui.fragments.main

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import app.editors.manager.BuildConfig
import app.editors.manager.R
import app.editors.manager.databinding.FragmentAboutBinding
import app.editors.manager.ui.fragments.base.BaseAppFragment
import lib.toolkit.base.managers.utils.FileUtils.readSdkVersion

class AboutFragment : BaseAppFragment(), View.OnClickListener {

    companion object {
        val TAG: String = AboutFragment::class.java.simpleName
        const val ASSETS_SDK_VERSION_PATH = "sdk.version"

        fun newInstance(): AboutFragment {
            return AboutFragment()
        }
    }

    private var viewBinding: FragmentAboutBinding? = null

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        viewBinding = FragmentAboutBinding.inflate(inflater, container, false)
        return viewBinding?.root
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        init()
    }

    override fun onDestroyView() {
        super.onDestroyView()
        viewBinding = null
    }

    private fun init() {
        setActionBarTitle(getString(R.string.about_title))
        val sdkVersion = readSdkVersion(requireContext(), ASSETS_SDK_VERSION_PATH)
        viewBinding?.aboutAppVersion?.text = getString(
            R.string.about_app_version,
            BuildConfig.VERSION_NAME,
            BuildConfig.VERSION_CODE.toString(),
            sdkVersion
        )
        viewBinding?.aboutTerms?.setOnClickListener(this)
        viewBinding?.aboutPolicy?.setOnClickListener(this)
        viewBinding?.aboutLicense?.setOnClickListener(this)
        viewBinding?.aboutWebsite?.setOnClickListener(this)
    }

    override fun onClick(view: View?) {
        when (view?.id) {
            R.id.about_terms -> showUrlInBrowser(getString(R.string.app_url_terms))
            R.id.about_policy -> showUrlInBrowser(getString(R.string.app_url_policy))
            R.id.about_license -> showFragment(LicenseFragment.newInstance(), LicenseFragment.TAG, false)
            R.id.about_website -> showUrlInBrowser(getString(R.string.app_url_main))
        }
    }
}